frontend/packages/kubevirt-plugin/locales/en/kubevirt-plugin.json

```diff
@@ -1051,6 +1051,12 @@
   "Interface with this name already exists": "Interface with this name already exists",
   "Network required)": "Network required)",
   "Invalid MAC address format": "Invalid MAC address format",
+  "Snapshot name cannot be empty": "Snapshot name cannot be empty",
+  "Snapshot name name can contain only alphanumeric characters": "Snapshot name name can contain only alphanumeric characters",
+  "Snapshot name must start/end with alphanumeric character": "Snapshot name must start/end with alphanumeric character",
```

Comment:
"character" should be plural like the "VM name..." line below.

Reply:
good catch fixed :)